docs/core/whats-new/dotnet-8.md

```diff
@@ -59,7 +63,9 @@ The serializer has built-in support for the following additional types.
 - <xref:System.Half>, <xref:System.Int128>, and <xref:System.UInt128> numeric types.
 
   ```csharp
-  Console.WriteLine(JsonSerializer.Serialize(new object[] { Half.MaxValue, Int128.MaxValue, UInt128.MaxValue }));
+  Console.WriteLine(JsonSerializer.Serialize(
+      new object[] { Half.MaxValue, Int128.MaxValue, UInt128.MaxValue }
+  ));
```

Comment:
Since this is .NET 8, let's use collection expressions:

```suggestion
  Console.WriteLine(JsonSerializer.Serialize(
      [ Half.MaxValue, Int128.MaxValue, UInt128.MaxValue ]
  ));
```